### ISSUE TITLE
Feature Request: Preview in column editor

### ISSUE THREAD

**Issue by BenBE** [htop-dev/htop#73]
When reconfiguring the list of displayed columns in the process view it's always somewhat cumbersome to add one column, exit the editor view, look if the display looks/feels like intended, reenter the editor and possibly do the next change.

To improve the UI in this respect it might be a nice idea to include a sample of 2-3 processes/threads at the bottom of the display as a preview of the selected columns and additionally (as space permits) sample values for the various columns you can select for inclusion.

Also for some columns (e.g. cgroup names) it might be good to introduce a somewhat "shortened" version (/sys/cgroup/lxc/container1 -> container1) to safe space.

**Comment by jb-boin** [comment]
I did make a patch for LXC/LXD in the previous repository : https://github.com/hishamhm/htop/issues/328

**Comment by BenBE** [comment]
I think these two issues are unrelated to the feature request.
Or am I missing something?

**Comment by fasterit** [comment]
@BenBE: Nope, you got that well. @jb-boin's comment only applies to his patch shortening strings as per the last line of your first comment above.
